.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
```suggestion
blank_issues_enabled: true
```

Suggest enable it.

Reply:
If this were to be enabled, people would have the ability to create blank issues without providing any formatting or guidance at all.

Reply:
Sometimes it's usefull, such as create a summary or a vote.